portfolio/src/main/webapp/style.css

```
@@ -264,6 +265,26 @@ input {
     display: none;
 }
 
+.error-alert {
+  background-color: #E05571;
+  border-radius: 3px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: space-between;
+  margin: 5px 0px 5px 0px; 
```

You can shorthand this to:

```suggestion
  margin: 5px 0; 
```

ref) https://www.w3schools.com/cssref/pr_margin.asp

---

Oh, awesome, didn't know about this! Done.